app/Main.hs

```diff
@@ -25,7 +25,7 @@ defaultProject =
           , projectFiles = []
           , projectAlreadyLoaded = []
           , projectEchoC = False
-          , projectLibDir = ".carp/libs/"
+          , projectLibDir = "carp/libs/"
```

Comment:
And here (plus the coming lines)?

Reply:
In this case I think it would be better to stick to unix paths and do a `toNative` where needed, since we don't want conditional compilation for something like configuration.

Or we could just use `System.FilePath.Posix` paths everywhere and perform the `toNative` before calling the OS. 

I'll try to evaluate which one looks better.